docs/get_started/getting_started_linux_cmake.md

```diff
@@ -107,6 +107,39 @@ $ ./build/iree/tools/iree-run-mlir $PWD/iree/tools/test/simple.mlir \
     -input-value="i32=-2" -iree-hal-target-backends=vmla -print-mlir
 ```
 
+### Dylib LLVM AOT backend
+
+To compile IREE LLVM AOT module we need to set the AOT linker path environment variable (eg: for lld-10):
```

Comment:
Let's add some more context here, at least spelling out "Ahead-of-Time (AOT)" the first time it's used in this document. We could also talk briefly (1-2 sentences) about what LLVM/DyLib/AOT are and how they compare to the other options in IREE (e.g. CPU backend compiling MLIR code through LLVM). I don't have more specific suggestions right now for how to phrase that.